source/upgrading/upgrade/upgrade_notes.rst

```diff
@@ -18,15 +18,28 @@ General Upgrade Notes
 =====================
 
 
-Java version upgraded to Java 1.8
+Java version upgraded to Java 1.11
 ---------------------------------
 
-As of Apache CloudStack 4.10, Java version required is 1.8 for the
+As of Apache CloudStack 4.14, Java version required is 1.11 for the
 management-server, cloudstack-usage, KVM agent and system-VMs.
 
 
-.. include:: _java_8_ubuntu.rst
+.. include:: _java_version.rst
 
+UI Deprecation Notice
```

Comment:
@rhtyd I don't see this whole section visible anywhere (in any of the specific version upgrade docs..) - I could include it myself somewhere, but not sure where is the best place - possibly in index page?

And we should probably also add that we don't support CentOS 6 anymore.

Reply:
There is a compatibility section in the release notes.